stdpopsim/drosophila_melanogaster.py

```diff
@@ -111,3 +111,47 @@ def __init__(self):
                 time=t_2, initial_size=N_A, population_id=0)
         ]
         self.migration_matrix = [[0]]
+
+
+class LiStephanTwoPopulation(models.Model):
+    """
+    two population model of Li and Stephan (2006) for
+    African and European divergence
+
+    .. todo:: document this model, including the original publications
+        and clear information about what the different population indexes
+        mean.
+
+    """
+
+    def __init__(self):
+
+        # African Parameter values from "Demographic History of the African
+        # Population" section
+        N_A0 = 8.603e06
+        t_A0 = 6000  # generations
+        N_A1 = N_A0 / 5.0
+        # European Parameter values from "Demo History of Euro Population"
+        N_E0 = 1.075e06
+        N_E1 = 2200
+        t_AE = 1580  # generations
+        t_E1 = 1580 - 34
+        # Single population in this model
```

Comment:
This comment is misleading. Best just delete it, as it's not telling us anything we can't see from the code.

Reply:
fixed now in the rebased commit

Reply:
I'm not sure what you did here, but it's not a rebase! I'm seeing 15 commits plus a couple of merges...  Did you follow the recipe? I can jump in to try to sort this out if you like.

Reply:
OK, it looks like you merged the branch into itself somehow, so if you drop the last commit, you should be OK. 

You need to force push after doing a rebase, ignore whatever git is telling you **after** doing a rebase.

Reply:
yeah i did-- fetch upstream; git rebase -i origin <branchname>. not sure what happened. maybe you can jump in and tell me what happened?

Reply:
Ah, you rebased on ``origin``, not ``upstream/master``.  It might sort itself out if you redo this: 
https://stdpopsim.readthedocs.io/en/latest/development.html#github-workflow

I do need to add a comment about ignoring what git says immediately after the rebase though, this is very confusing and misleading...

Reply:
btw this is what i get trying to rebase code with my kids hanging on me! sorry to screw this up

Reply:
I had a quick go at rebasing this, and it's a mess. The easiest thing to do is to get a patch:
```
$ git diff upstream/master > andy.patch
$ git reset --hard upstream/master
$ patch -p1 < andy.patch
$ git commit -a -m "added Li and Stephan model, changes to example"
$ git push -f origin li_stephan_model
```

Reply:
will give it a go!

Reply:
seems to have worked!!!